src/app.js

```diff
@@ -330,5 +302,18 @@ export default function (app) {
 
     return element
   }
+
+  function event(type) {
+    var args = Array.prototype.slice.call(arguments, 1);
+    (subscriptions[type] || []).forEach(function(cb) { cb.apply(null, args) });
+    (subscriptions["*"] || []).forEach(function(cb) { cb.apply(null, type) });
```

Comment:
That's a catch all for events. If you register a listener to `*` it will be called whenever an event or hook is called.